### PR TITLE
update drivetime band client and configuration namespacing

### PR DESCRIPTION
**Pull request by LindseySaari**
## Description of change
This change addresses namespacing cleanup within the facilities drivetime bands directory (lib/facilities/drivetime_bands). In order to better understand separation of facility services, namespacing improvements were introduced.

It was decided to break the original PR up into smaller PRs. Link to Original PR: https://github.com/department-of-veterans-affairs/vets-api/pull/3565

## Testing done
Test suite passing

#### Unique to this PR
- [x] Passing test suite

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: lib/facilities/drivetime_bands/configuration.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+module Facilities
+  module DrivetimeBands
+    class Configuration < Common::Client::Configuration::REST
+      def base_path
+        Settings.locators.drive_time_band_base_path
+      end
+
+      def service_name
+        'FL'
```
Comment (johnpaulashenfelter) [inline lib/facilities/drivetime_bands/configuration.rb]: Just curious -- where did that name/abbreviation come from?

Comment (LindseySaari) [reply lib/facilities/drivetime_bands/configuration.rb]: I think it might stand for Facility Locator?

Comment (LindseySaari) [reply lib/facilities/drivetime_bands/configuration.rb]: There's a job that does some Geo coding on drivetime bands for facilities

Comment (nfasulo) [reply lib/facilities/drivetime_bands/configuration.rb]: When we were creating this we decided to stick with "FL" to match the existing facilities services, but would be happy to see it replaced with a more descriptive name at some point.

**Pull request merged**